transport/http2_server.go

```diff
@@ -863,6 +884,7 @@ func (t *http2Server) Write(s *Stream, data []byte, opts *Options) (err error) {
 func (t *http2Server) applySettings(ss []http2.Setting) {
 	for _, s := range ss {
 		if s.ID == http2.SettingInitialWindowSize {
+			atomic.AddUint64(&t.outQuotaVersion, 1)
```

Comment:
Yes, InitialWindowSize settings frame can be sent multiple times during the  life of a connection. In fact, that's why we're seeing this error. C decreases the IWS during it's bdp calculations causing the issue to occur.